src/ext/sse.js

```diff
@@ -196,6 +196,17 @@ This extension adds support for Server Sent Events to htmx.  See /www/extensions
 				}
 			}
 		});
+
+		var closeAttribute = api.getAttributeValue(elt, "sse-close");
+		if (closeAttribute) {
+			// close eventsource when this message is received
+			var source = api.getInternalData(elt).sseEventSource;
+			if (source != undefined) {
+				source.addEventListener(closeAttribute, function() {
+					source.close()
```

Comment:
Should also invoke `maybeCloseSSESource` like other event handlers

Reply:
I'm unsure about how to implement this change. The other functions that call this seem to all use swap, but since I'm not adding an additional element it appears that `maybeCloseSSESource()` finds the element in the body and thus doesn't close the source. Should I manufacture an extra element when setting up the event listener? Or perhaps I'm just passing the wrong element to search for in `maybeCloseSSESource()`?

Reply:
Thinking about it, it was a silly suggestion on my side. When this event handler is triggered, the event source will be closed anyway, there's no need to double check. Sorry 😅